mozjs-sys/makefile.cargo

```diff
@@ -148,12 +148,18 @@ all: maybe-configure
 # The second two time checks handle the case of configure.in and configure having
 # the same timestamp (e.g. after a git checkout)
 JSSRC := '$(SRC_DIR)'/js/src
+# Keep track of all input variables to configure, so we don't skip reconfigure
+# when we do need to reconfigure
+CONFIGURE_INPUTS := "$(CC)$(CFLAGS)$(CPP)$(CPPFLAGS)$(CXX)$(CXXFLAGS)$(AS)$(AR)$(CONFIGURE_FLAGS)"
+LAST_CONFIGURE_INPUTS := "$(shell cat reconfigure.inputs)"
```

Comment:
what is shell command? It's not available in git bash.

Reply:
isn't that make syntax to call the shell (bash)? Just doing `$(cat reconfigure.inputs)` did not work for me.

Reply:
right, this is makefile not bash script!